bagel/dictionary_models.py

```diff
@@ -66,7 +88,7 @@ class ContinuousNeurobagel(Neurobagel):
 class IdentifierNeurobagel(Neurobagel):
     """A Neurobagel annotation for an identifier column"""
 
-    identifies: "str" = Field(
+    identifies: str = Field(
```

Comment:
was that just something we missed or is that actually pydantic v2 related?

Reply:
just something we missed, I think. The old syntax doesn't seem to actually break anything (I think Pydantic may be clever enough to infer the type?) but it does look weird